controller/src/beerocks/cli/beerocks_cli_main.cpp

```diff
@@ -486,10 +483,6 @@ int main(int argc, char *argv[])
             cli_role       = CLI_NON_INTERACTIVE;
             break;
         }
-        case 'w': {
```

Comment:
Please add to the commit message why did you remove the wait option.

Reply:
> Please add to the commit message why did you remove the wait option.

Whatever the reason: it should probably be a separate commit anyway.

Reply:
Yes, I meant to do this yesterday, I'll do this now.

Reply:
Fixed.